src/app/components/character-card/character-card.component.scss

```diff
@@ -0,0 +1,95 @@
+@import '/src/scss/colors';
+
+.card {
+  align-items: flex-start;
+  border: 1px solid $disabled;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 350px;
+  justify-content: space-around;
+  max-width: 250px;
+  padding: 8px 15px;
+  position: relative;
```

Comment:
Why was this style added?

Reply:
This is the card content distribution, and the relative position is to control the check-box that is absolute.